build.sbt

```diff
@@ -46,6 +46,7 @@ lazy val tests = crossProject(JVMPlatform, NativePlatform)
   .enablePlugins(NoPublishPlugin)
   .nativeConfigure(_.dependsOn(core))
   .settings(
+    Test / testOptions += Tests.Argument("+l"),
```

Comment:
Any idea what the +ell means? I spent some time on the Web looking for it
in both the sbt and munit docs. Seem like kind of a "If this person gets hit by a bus..."
maintenance situation.

Is it something inherited from PR which provided the template for this PR?
That is understandable & far better than the quirks before.

Reply:
Yes I had to do quite a bit of digging as well. It comes from here:
https://github.com/scalameta/munit/blob/92710a507339d20368d251feadf66e4f9f4e1840/junit-interface/src/main/java/munit/internal/junitinterface/JUnitRunner.java#L64-L102

I believe it is equivalent to setting `--logger=sbt`. So I will switch to use that and add a comment linking to the above.

Reply:
Yes!  

It is a  delicate balancing act between up-front actually-spent costs spent now versus
envisioned downstream maintenance costs  maybe/maybe_not spent by somebody else.

In an open-source  project which hopes to attract additional developers, having a 
good model/template has value.

In my mind you strike a good balance.

Reply:
Amazingly, `--logger=sbt` does not work ... no idea at this point 😂 so I just added plenty of comments, and kept this current solution.